packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -243,6 +262,7 @@ export class VaMemorableDate {
             class='usa-form-group--month-select'
             reflectInputError={error ? true : false}
             value={month ? String(parseInt(month)) : month}
+            ref={(field) => this.monthfield = field}
```

Comment:
I'm not sure that I understand why we need `ref` on the fields. What does this do different than just querySelect'ing the field directly with a class or id? (ie. `this.el.shadowRoot.querySelector`)

Reply:
I think I understand why now. Is this so that you can apply focus?

Reply:
My logic around this is that the reference to the fields is stored once on render, whereas the querySelector would need to query the DOM each time validation is ran. 

I don't know if `ref` is a stencil/react thing and not a thing with native web components? If that is the case maybe we should change it to use querySelector.

Reply:
I agree. That change has been made.